projects/packages/my-jetpack/_inc/components/product-cards-section/boost-card/boost-speed-score.tsx

```diff
@@ -5,23 +5,28 @@ import { IconTooltip, Spinner, BoostScoreBar } from '@automattic/jetpack-compone
 import { __, sprintf } from '@wordpress/i18n';
 import React, { useCallback, useEffect, useState } from 'react';
 import useAnalytics from '../../../hooks/use-analytics';
+import useMyJetpackConnection from '../../../hooks/use-my-jetpack-connection';
 import './style.scss';
 
 const BoostSpeedScore = () => {
 	const { recordEvent } = useAnalytics();
 	const [ isLoading, setIsLoading ] = useState( false );
-	const [ speedLetterGrade, setSpeedLetterGrade ] = useState( 'C' );
+	const [ speedLetterGrade, setSpeedLetterGrade ] = useState( '' );
 	const [ daysSinceTested, setDaysSinceTested ] = useState( 1 );
 	const [ averageSpeedScore, setAverageSpeedScore ] = useState( 0 );
 	const [ isSpeedScoreError, setIsSpeedScoreError ] = useState( false );
 
-	const { apiRoot, apiNonce, connectionStatus } = window.JP_CONNECTION_INITIAL_STATE;
+	const { apiRoot, apiNonce } = window.JP_CONNECTION_INITIAL_STATE;
```

Comment:
I've also seen apiRoot and apiNonce extracted from useMyJetpackConnection. Here is an example: https://github.com/Automattic/jetpack/blob/trunk/projects/packages/my-jetpack/_inc/components/connections-section/index.jsx#L14

Reply:
Ahhh right.  Yes, definitely, Thanks! 👍